crates/libcontainer/src/rootfs/utils.rs

```diff
@@ -118,7 +159,12 @@ pub fn parse_mount(m: &Mount) -> MountOptionConfig {
                 "norelatime" => Some((true, MsFlags::MS_RELATIME)),
                 "strictatime" => Some((true, MsFlags::MS_STRICTATIME)),
                 "nostrictatime" => Some((true, MsFlags::MS_STRICTATIME)),
-                _ => None,
+                unknown => {
```

Comment:
Wouldn't this silently ignore unknown mount options that are not idmap? I think we should always return an error here.

Reply:
For example, `mode=755` should be passed these codes, but we don't need to set `MsFlags`.
https://github.com/opencontainers/runtime-spec/blob/main/config.md#linux-mount-options
> Runtimes SHOULD pass unknown options to [mount(2)](http://man7.org/linux/man-pages/man2/mount.2.html) via the fifth argument (const void *data).

`idmap` and `ridmap` are defined in OCI Runtime Spec, So users may think these options will affect their mounts and pass them on. But we do not support them and would like to return an error to be safe in this case.